G1ANT.Addon.UI/Api/UIElement.cs

```diff
@@ -158,7 +186,15 @@ public void Click()
 
         public void SetFocus()
         {
+
             automationElement.Focus();
+            var currentFocusedElement = AutomationSingleton.Automation.FocusedElement();
+            if (!automationElement.Equals(currentFocusedElement))
+            {
+                var parentWindow = automationElement.GetParentElementClosestToDesktopElement();
+                parentWindow?.Focus();
+                automationElement.Focus();                
```

Comment:
better use if (paretnWindow != null) {} and invoke automationElement.Focus() only then